system/Helpers/array_helper.php

```diff
@@ -122,3 +122,81 @@ function array_deep_search($key, array $array)
 		return null;
 	}
 }
+
+if (!function_exists('sort_by_multiple_keys'))
+{
+	/**
+	 * Sorts an associative multidimensional array by the values of the
+	 * passed columns names. The sorting params represent an associative
+	 * array of column names and sorting flags.
+	 * Both arrays of objects and arrays of array can be sorted.
```

Comment:
```suggestion
	 * Sorts an associative multidimensional array by the values of the
	 * passed columns' names. The sorting params represent an associative
	 * array of column names and sorting flags.
	 *
	 * Both array of objects and array of arrays can be sorted.
```

Reply:
I rewrote this section, sounded a little confusing reading it again.